src/Components/Calculator.js

```diff
@@ -15,30 +27,32 @@ class Calculator extends React.Component {
               <p id="history-value" />
             </div>
             <div id="output">
-              <p id="output-value">0</p>
+              {total}
+              {operation}
+              {next}
             </div>
           </div>
           <div id="keyboard">
-            <button type="button" className="operator" id="clear">C</button>
-            <button type="button" className="operator" id="backspace">AC</button>
-            <button type="button" className="operator" id="backspace">+/-</button>
-            <button type="button" className="operator" id="+">&#247;</button>
-            <button type="button" className="operator" id="/">7</button>
-            <button type="button" className="number" id="7">8</button>
-            <button type="button" className="number" id="8">9</button>
-            <button type="button" className="number" id="9">&times;</button>
-            <button type="button" className="operator" id="*">4</button>
-            <button type="button" className="number" id="4">5</button>
-            <button type="button" className="number" id="5">6</button>
-            <button type="button" className="number" id="6">-</button>
-            <button type="button" className="operator" id="-">1</button>
-            <button type="button" className="number" id="1">2</button>
-            <button type="button" className="number" id="2">3</button>
-            <button type="button" className="number" id="3">+</button>
-            <button type="button" className="operator" id="%">%</button>
-            <button type="button" className="empty" id="empty">0</button>
-            <button type="button" className="number" id="0">.</button>
-            <button type="button" className="operator" id="=">=</button>
+            <button type="button" className="operator" id="clear" onClick={this.handleBtnClick}>C</button>
+            <button type="button" className="operator" id="backspace" onClick={this.handleBtnClick}>AC</button>
+            <button type="button" className="operator" id="backspace" onClick={this.handleBtnClick}>+/-</button>
+            <button type="button" className="operator" id="+" onClick={this.handleBtnClick}>&#247;</button>
+            <button type="button" className="operator" id="7" onClick={this.handleBtnClick}>7</button>
+            <button type="button" className="number" id="8" onClick={this.handleBtnClick}>8</button>
+            <button type="button" className="number" id="9" onClick={this.handleBtnClick}>9</button>
+            <button type="button" className="number" id="*" onClick={this.handleBtnClick}>&times;</button>
```

Comment:
![image](https://user-images.githubusercontent.com/66112760/148263833-819aee36-7b95-408a-8ce5-df3a5e0afc61.png)

Kindly fix this error, as the web page goes blank when I use the '*' operator sign